packages/typescript-estree/tests/lib/convert.test.ts

```diff
@@ -29,7 +29,6 @@ describe('convert', () => {
       function fakeUnknownKind(node: ts.Node): void {
         ts.forEachChild(node, fakeUnknownKind);
         // @ts-expect-error -- intentionally writing to a readonly field
-        // eslint-disable-next-line deprecation/deprecation, @typescript-eslint/no-unused-expressions
```

Comment:
re: `deprecation/deprecation`, `ts.SyntaxKind` has actually been removed from TypeScript altogether (!) — this line is not throwing any errors, though, because of the unrelated `ts-expect-error`

- `fakeUnknownKind` was originally added in #1525 
- `deprecation/deprecation` suppression added in #6570 
- `TSUnparsedPrologue` removed from snapshots (and TypeScript completely) in #9397